setup.py

```diff
@@ -38,6 +38,7 @@
     'pytest>=3.4.2',
     'pytest-cov>=2.6.0',
     'pytest-rerunfailures>10',
+    'nbconvert>5,<6.5.0',
```

Comment:
do we need this?

Reply:
Not really, since I couldn't reproduce it.